cmd/minikube/cmd/start.go

```diff
@@ -172,6 +173,13 @@ func runStart(cmd *cobra.Command, args []string) {
 	console.OutStyle(console.Happy, "minikube %s on %s (%s)", version.GetVersion(), runtime.GOOS, runtime.GOARCH)
 	validateConfig()
 
+	currentUser, err := user.Current()
+
+	// Display warning if minikube is being started with root and vmDriver is not hyperv
+	if err == nil && currentUser.Name == "root" && viper.GetString(vmDriver) != "hyperv" {
```

Comment:
could you check if this works under windows too ? is it called "root" or "admin" or "adminstrator" in windows?

Reply:
Using sudo (or running as root) is required, when using the `none` driver (as a special case).